.travis.yml

```diff
@@ -19,7 +19,7 @@ matrix:
     - env: TOX_ENV=check
       python: 3.6
 install: pip install -U tox
-script: tox -e $TOX_ENV
+script: travis_retry tox -e $TOX_ENV
```

Comment:
We may also wanna upgrade `dist: precise` as it has been deprecated. but i recall that we have a unit test failed on `dist: trusty` ?

https://blog.travis-ci.com/2017-08-31-trusty-as-default-status

Reply:
Ya, it's still failing with trusty. Looks like it just stops responding: https://travis-ci.org/channelcat/sanic/jobs/349989523